src/main/java/Task/DukeException.java

```diff
@@ -0,0 +1,5 @@
+package Task;
+
+public class DukeException extends Exception {
+
```

Comment:
Here, use the opportunity provided by inheritance to include a more expressive error message rather than just create a blank class